### PR TITLE
Update logging.rst

### PR DESCRIPTION
**Pull request by SychevIgor**
Links to source code on a GitHub updated to current version rc1.

### CODE REVIEW

**Comment by SychevIgor** [comment]
Looks like project have some issues with CI. Because links are valid.

File: aspnet/fundamentals/logging.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ The console logger prefixes verbose output with "verbose: " and uses a gray font
...
+In the course of logging information within your application, you can group a set of logical operations within a *scope*. A scope is an ``IDisposable`` type returned by calling the ``BeginScopeImpl`` method, which lasts from the moment it is created until it is disposed. Not all loggers support scopes. For example, the `ConsoleLogger simply returns null <https://github.com/aspnet/Logging/blob/1.0.0-beta7/src/Microsoft.Framework.Logging.Console/ConsoleLogger.cs#L114>`_ from ``BeginScopeImpl``. The built-in `TraceSource logger <https://github.com/aspnet/Logging/blob/1.0.0-rc1/src/Microsoft.Extensions.Logging.TraceSource/TraceSourceLogger.cs#L66-L69>`_ returns a scope instance that is responsible for starting and stopping tracing operations. Any logging state, such as a transaction id, is attached to the scope when it is created.
```
Comment (shirhatti) [inline aspnet/fundamentals/logging.rst]: @kichalla Could you have a look at this?

Comment (kichalla) [inline aspnet/fundamentals/logging.rst]: @shirhatti Starting with RC1, ConsoleLogger also supports scopes...a user can look this information by doing `loggerFactory.AddConsole(includeScopes: true)`

Also the link for ConsoleLogger should point to `rc1`

**Comment by SychevIgor** [comment]
Please, see this pull request https://github.com/aspnet/Docs/pull/732

**Pull request closed without merging**